### ISSUE TITLE
Installation / compile fails due to breaking git2 changes

### ISSUE THREAD

**Issue by beevelop** [saschagrunert/git-journal#6]
Installing the current version of git-journal results in the following exceptions being thrown:
```
error[E0425]: cannot find value `SORT_TIME` in module `git2`
error[E0425]: cannot find value `REVPARSE_SINGLE` in module `git2`
error[E0425]: cannot find value `REVPARSE_MERGE_BASE` in module `git2`
```

It seems as if those variables have been removed in git2 `v0.7.0` onwards. Unfortunately the git2 versions seems not be pinned to the latest `0.6` release.

**Comment by saschagrunert** [comment]
Hey thank you very much for your report. I will investigate this as soon as I am able to.

**Comment by beevelop** [comment]
Thank you for the fast response. I tested swapping the version specification in the `Cargo.toml` locally. Pinning it to `0.6` seems to prevent the installation from failing (not a solid fix, but maybe a suitable monkey-patch).

**Comment by saschagrunert** [comment]
Yeah if this helps for a first shot we can go with 0.6.x. Feel free to submit a pull request. I will investigate further these days... :)

**Comment by saschagrunert** [comment]
This issue should be fixed with version 1.6.3.

**Comment by beevelop** [comment]
@saschagrunert Awesome, thank you for the fix 🍻
